### DISCUSSION TITLE
Blurry quality

### DISCUSSION THREAD

**Discussion by metagamesgx** [WyattBlue/auto-editor#604] [1. Help] [answered]
Hello when rendering using the GUI I have very blurry quality on the video that comes out. Is there anyway to render in original quality (lossless) so that the video quality isnt less than what it came in as?

**Comment by gaurav-kr-pandey** [comment]
facing same issue

**Comment by Rael211** [comment]
Same here. Much lower quality of the output.

**Comment by WyattBlue** [comment, marked as answer]
One loseless video format is huffyuv, although it will produce hilariously large files.
```
auto-editor input.mp4 -c:v huffyuv
```

A better method is to use a near loseless encoder like `vp9` or `hevc`
```
auto-editor input.mp4 -c:v vp9
```

If you have to use a particular video format, you can increase the bitrate, in this case to 2 megabytes
```
auto-editor input.mp4 -b:v 2M
```

**Comment by Rael211** [comment]
Thank you! Will try.
  
Orlin
  

  

  
  
--   
  
Best regards:   
  
Orlin Chotev   
  
tel. 781 346 9844
  

        
  
  
>
